test/functional/wallet_importdescriptors.py

```diff
@@ -0,0 +1,293 @@
+#!/usr/bin/env python3
+# Copyright (c) 2019 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test the importdescriptors RPC.
+
```

Comment:
this test needs to check default arguments like `active`

Reply:
How so?

Reply:
Make a test to ensure that `active` is actually default true(or not)

Reply:
Done